tests/test_wfs.py

```diff
@@ -254,15 +270,17 @@ def test_geopandas(self):
                 )
             )
 
-            self.assertEqual(len(m.request_history), 3)
+            self.assertEqual(len(m.request_history), 4)
 
             workflow_request = m.request_history[1]
             self.assertEqual(workflow_request.method, "POST")
             self.assertEqual(workflow_request.url,
                              "http://mock-instance/workflow")
             self.assertEqual(workflow_request.json(), workflow_definition)
 
-            wfs_request = m.request_history[2]
+            # second request is result descriptor retrieval
```

Comment:
I find this comment confusing

suggestion: "# note: result descriptor is automatically retrieved upon workflow registration (Workflow constructor), thus the actual wfs request is in the third history slot"

and shouldn't it be "third request" if the index is "2"?

Reply:
cdb554b